nsqd/http.go

```diff
@@ -99,11 +100,23 @@ func newHTTPServer(ctx *context, tlsEnabled bool, tlsRequired bool) *httpServer
 	router.Handler("GET", "/debug/pprof/heap", pprof.Handler("heap"))
 	router.Handler("GET", "/debug/pprof/goroutine", pprof.Handler("goroutine"))
 	router.Handler("GET", "/debug/pprof/block", pprof.Handler("block"))
+	router.HandlerFunc("PUT", "/debug/blockrate",
+		HandleBlockRate)
 	router.Handler("GET", "/debug/pprof/threadcreate", pprof.Handler("threadcreate"))
 
 	return s
 }
 
+func HandleBlockRate(w http.ResponseWriter, req *http.Request) {
+	rate, err := strconv.Atoi(req.FormValue("rate"))
+	if err != nil {
+		w.WriteHeader(http.StatusBadRequest)
+		fmt.Fprintf(w, "block rate value error: %s", err.Error())
+		return
+	}
+	runtime.SetBlockProfileRate(rate)
```

Comment:
Since the SetBlockProfileRate has no return value, I think we no need to return output.